buddyboss/buddyboss-platform
# Performance > Duplicate queries for @mentions when rendering activity feed - Fixed

### All Submissions:






### Changes proposed in this Pull Request:

Prevented initiating new activity object(BP_Activity_Activity) if the activity object is already available in the global activity template.

Fixes #3070 .

### How to test the changes in this Pull Request:

1. Create an activity post with multiple @mentions
2. Capture the queries for the activity item where has multiple @mentions when rendering.
3. See no duplicate queries

### Proof Screenshots or Video

https://www.loom.com/share/0a90b9669792476c8a0fa3245177a30b

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
